src/components/OfflineWithFeedback.tsx

```diff
@@ -83,7 +78,7 @@ function OfflineWithFeedback({
 
     const hasErrors = isNotEmptyObject(errors ?? {});
     // Some errors have a null message. This is used to apply opacity only and to avoid showing redundant messages.
-    const errorMessages = omitBy(errors, (e) => e === null);
+    const errorMessages = Object.fromEntries(Object.entries(errors ?? {}).filter((errorEntry): errorEntry is [string, string] => errorEntry[1] !== null));
```

Comment:
I'll try to improve it, the main reason why it was changed is that errors can have a `null` message (you can even see it in the comment above in line 80). Updated code narrow down the types to identify that updated `errorMessages` object doesn't have `null` values